src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -15,15 +22,17 @@ const ScenarioSidebar: React.FC = () => {
           </p>
         </div>
         <div>
-          <p className="text-xs text-gray-500">Last Update: March 25, 2020</p>
+          <p className="text-xs text-gray-500">Last Update:</p>
         </div>
         <div className="mt-5 mb-5 border-b border-gray-300" />
         <div>
           <ToggleRow
+            data={dailyReports}
             label="Daily Reports"
             labelHelp="Tooltip help Lorem ipsum dolor sit amet, consectetur adipiscing elit"
```

Comment:
We should get final copy for all of the "lorem ipsum" text in the side bar as well

Reply:
I'm going to take a pass through and update all copy this weekend!